jetty-fcgi/fcgi-client/src/main/java/org/eclipse/jetty/fcgi/parser/Parser.java

```diff
@@ -89,9 +106,12 @@ public boolean parse(ByteBuffer buffer)
                             // parsing; the async operation will eventually resume parsing.
                             return true;
                         }
+                        else
```

Comment:
There is no need for an `else` block here as that branch is the fall-though one.

Reply:
Done.